emails/notify_patron.spt

```diff
@@ -0,0 +1,19 @@
+{{ _("{0} from {1} has joined Gratipay!", user_name, platform) }}
+
+[---] text/html
+{{ _("Your pledge to give {0} every week to {1} will be turned into action now that they have joined Gratipay.",
+     format_currency(amount, 'USD'), user_name) }}
+<br><br>
+<a href="{{ profile_url }}" style="color: #fff; text-decoration:none; display:inline-block; padding: 0 15px; background: #396; font: normal 14px/40px Arial, sans-serif; white-space: nowrap; border-radius: 3px">{{ _("View {0}'s profile", user_name) }}</a>
```

Comment:
Can we abstract the button styles in emails? I'd like to be able to tell by looking at this code whether it'll result in the same button we use for the verification email.